sdcm/utils/gce_utils.py

```diff
@@ -541,11 +541,7 @@ def create_instance(  # noqa: PLR0913
     instance.scheduling = compute_v1.Scheduling()
 
     if "z3-highmem" in machine_type:
-        instance.scheduling.on_host_maintenance = "TERMINATE"
-        network_interface.nic_type = "GVNIC"
-        network_performance_config = compute_v1.NetworkPerformanceConfig()
-        network_performance_config.total_egress_bandwidth_tier = "TIER_1"
-        instance.network_performance_config = network_performance_config
+        instance.scheduling.on_host_maintenance = "MIGRATE"
```

Comment:
this now works for all of z3 instances ?

Reply:
I think at some point we'll be able to drop it (let's leave a comment to check it after release)

Reply:
it's controlling on how google doesn't maintenance to nodes.
and during preview, they usually treat is a bit differently.

we don't really care, since we don't enable automatic maintenance on our nodes (and if such is needed, it would break our test anyhow)

if you want to read more about it:
https://cloud.google.com/compute/docs/instances/host-maintenance-overview#schedulingoptions